Add CSS -eos-cairo-filter property patch

xjuan opened the pull request:
This property is used to provide a hint about the algorithm it should be use to scale CSS images.

https://phabricator.endlessm.com/T20677

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

Commit:
Add CSS -eos-cairo-filter property patch

This property is used to provide a hint about the algorithm it should be use to scale CSS images.

https://phabricator.endlessm.com/T20677
